replicated-security/baryon-1/baryon-1-genesis-without-ccv.json

```diff
@@ -0,0 +1,291 @@
+        "list": []
+      },
+      "last_transmission_block_height": {
+        "height": "0"
+      }
+    },
+    "contractmanager": {
+      "params": {},
+      "failures_list": []
+    },
+    "crisis": {
+      "constant_fee": {
+        "denom": "untrn",
+        "amount": "1000"
+      }
+    },
+    "evidence": {
+      "evidence": []
+    },
+    "feeburner": {
+      "params": {
+        "neutron_denom": "untrn",
+        "treasury_address": "neutron1vguuxez2h5ekltfj9gjd62fs5k4rl2zy5hfrncasykzw08rezpfsd2rhm7"
+      },
+      "total_burned_neutrons_amount": {
+        "coin": {
+          "denom": "",
+          "amount": "0"
+        }
+      }
+    },
+    "feegrant": {
+      "allowances": []
+    },
+    "feerefunder": {
+      "params": {
+        "min_fee": {
+          "recv_fee": [],
+          "ack_fee": [
+            {
+              "denom": "untrn",
+              "amount": "1000"
+            }
+          ],
+          "timeout_fee": [
+            {
+              "denom": "untrn",
+              "amount": "1000"
+            }
+          ]
+        }
+      },
+      "fee_infos": []
+    },
+    "ibc": {
+      "client_genesis": {
+        "clients": [],
+        "clients_consensus": [],
+        "clients_metadata": [],
+        "params": {
+          "allowed_clients": [
+            "06-solomachine",
+            "07-tendermint"
+          ]
+        },
+        "create_localhost": false,
+        "next_client_sequence": "0"
+      },
+      "connection_genesis": {
+        "connections": [],
+        "client_connection_paths": [],
+        "next_connection_sequence": "0",
+        "params": {
+          "max_expected_time_per_block": "30000000000"
+        }
+      },
+      "channel_genesis": {
+        "channels": [],
+        "acknowledgements": [],
+        "commitments": [],
+        "receipts": [],
+        "send_sequences": [],
+        "recv_sequences": [],
+        "ack_sequences": [],
+        "next_channel_sequence": "0"
+      }
+    },
+    "interchainaccounts": {
+      "controller_genesis_state": {
+        "active_channels": [],
+        "interchain_accounts": [],
+        "ports": [],
+        "params": {
+          "controller_enabled": true
+        }
+      },
+      "host_genesis_state": {
+        "active_channels": [],
+        "interchain_accounts": [],
+        "port": "icahost",
+        "params": {
+          "host_enabled": true,
+          "allow_messages": []
+        }
+      }
+    },
+    "interchainqueries": {
+      "params": {
+        "query_submit_timeout": "19938",
+        "query_deposit": [
+          {
+            "denom": "untrn",
+            "amount": "1000000"
+          }
+        ]
+      },
+      "registered_queries": []
+    },
+    "interchaintxs": {
+      "params": {
+        "msg_submit_tx_max_messages": "16"
+      }
+    },
+    "params": null,
+    "slashing": {
+      "params": {
+        "signed_blocks_window": "10000",
```

Comment:
What is the average block time you are aiming for? For a 5s average block time and a `min_signed_per_window` of 0.5,  validators would have about 7h to join after the first block.

Reply:
good catch